data/downloadsfinanzierung.yml

```diff
@@ -20,10 +27,10 @@
     - 2020
 - de:
     title: Bericht der Wirtschaftsprüfung
-    text: Die vollständige Liste der Wirtschaftsprüfungen seit ihrer Einführung 2018 findest du unter Transparenz.
+    text: Die vollständige Liste der Wirtschaftsprüfungen seit ihrer Einführung 2018 findest du unter [Transparenz](/transparenz).
   en:
     title: Audit Reports [DE]
```

Comment:
change to: 

`title: Audit Reports`

Reply:
comment: remove [DE] as it is redundant, all other documents are in german as well